hawkular-alerts-engine/src/main/resources/org/hawkular/alerts/engine/rules/ConditionMatch.drl

```diff
@@ -378,7 +379,37 @@ rule AlertOnSatisfiedDampening
                 notifications.send(notification);
             }
         }
-        insert(  newAlert );
+
+        //insert(  newAlert );
+
         $d.reset();
         update( $d )
+
+        if ($t.isSafetyEnabled()) {
+            if (log != null && log.isDebugEnabled()) {
+                log.debug("Setting Trigger to Safety Mode! " + $t);
+            }
+            $t.setMode(Mode.SAFETY);
+            update( $t )

```

Comment:
At this point we need to re-feed the DefinitionsService in some way.
When the Trigger is modified, from the API, the UI should take account of this change, so I guess we need to implement this flow, as the safety mode can be set up from the UI, right ?